deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -94,10 +94,7 @@ spec:
                 version:
                   type: string
               required:
-              - group
```

Comment:
@baijum Why did you remove `gvr` from required?

Reply:
Reverted. 83750c1b4731363693d85af45ad30ffb89027f78

I am not sure why the test is failing this was just an experiment.

Reply:
ok